pkg/daemon/daemon.go

```diff
@@ -12,10 +12,10 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/fluxcd/flux/pkg/api"
-	"github.com/fluxcd/flux/pkg/api/v10"
-	"github.com/fluxcd/flux/pkg/api/v11"
-	"github.com/fluxcd/flux/pkg/api/v6"
-	"github.com/fluxcd/flux/pkg/api/v9"
+	v10 "github.com/fluxcd/flux/pkg/api/v10"
```

Comment:
Was this (and the ff) changed by an IDE? I don't think it's necessary, or relevant to this PR.

Reply:
Removed. VS Code + the Go extension was making this change on save.